src/DropDownMenu/DropDownMenu.js

```diff
@@ -28,7 +32,8 @@ function getStyles(props, context) {
     icon: {
       fill: accentColor,
       position: 'absolute',
-      right: spacing.desktopGutterLess,
+      right: isRtl ? '' : iconPadding,
+      left: isRtl ? iconPadding : '',
```

Comment:
Shall I consider moving some RTL styling logic in this PR to rtl.js, such as swapping right and left padding values?